public/data/sachdevlaksh.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Lakshay Sachdeva",
+  "type": "personal",
+  "bio": "Programmer | Backend Development | Microservices | Golang | Blockchain",
+  "avatar": "https://github.com/sachdevlaksh.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/sachdevlaksh",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/sachdevlaksh92",
+      "icon": "twitter"
+    },
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/lakshaysachdeva/",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "First interactions with PC",
+      "date": "January 2005",
+      "icon": "search",
+      "color": "#dadbe7",
+      "description": "Unenumerated hours of playing games on computer."
+    },
+    {
+      "title": "Joined Dual Degree B.Tech + M.Tech course in Electronics + VLSI. ",
+      "date": "July 2010",
+      "icon": "graduation-hat",
+      "color": "#699cf7",
+      "description": "Moved to Jaipur Rajasthan from Haryana"
+    },
+    {
+      "title": "First line of code!",
+      "date": "March 2011",
+      "icon": "book",
+      "color": "#dadbe7",
+      "description": "At age 20 got book about Ansi C, started coding simple hello world programs. "
```

Comment:
```suggestion
      "description": "At age 20 got book about Ansi C, started coding simple hello world programs."
```

Reply:
@schmelto. Thanks for review comment, I will change.